minard/templates/burst.html

```diff
@@ -6,50 +6,115 @@
-                  <tr>
-                      <td><a href="{{ url_for('burst_run_detail', run_number=run[1]['run'], subrun=run[1]['subrun'], sub=run[1]['burst_n']) }}">{{ run[1]['run'] }}</a></td>
-                      <td>{{  run[1]["subrun"] }}</td>
-                      <td>{{  run[1]["burst_n"] }}</td>
-                      <td>{{  run[1]["date"] }}</td>
-                      <td>{{  run[1]["time"] }}</td>
-                      <td>{{  run[1]["zdab_file"] }}</td>
-                      <td>{{  run[1]["burst_size"] }}</td>
-                      <td>{{  run[1]["burst_length"] }}</td>
-                      <td>{{  run[1]["avg_nhit"] }}</td>
-                  </tr>
-                  {% endfor %}
-              </tbody>
-          </table>
-          {% if offset > limit-1 %}
-            <a href="{{ url_for("burst", offset=offset-limit, limit=limit) }}">Back</a>
-          {% endif %}
-          {% if total > offset + limit %}
-          <a href="{{ url_for("burst", offset=offset+limit, limit=limit) }}">Next</a>
-          {% endif %}
+            <div class="row">
+                <div class="col-md-12">
+                  <div class="panel panel-default">
+                      <div class="panel-heading">
+                          Search:
+                      </div>
+                      <div class="panel-body">
+                          <form class="form-horizontal" role="form" method="get" action="{{ url_for("burst") }}" onsubmit="return validate_form()">
+                              <div class="form-group form-group-sm">
+                                  <label class="col-sm-2 control-label">
+                                      Search by:
+                                  </label>
+                                  <div class="col-sm-2">
+                                      <select id="search-by" name="search" class="form-control" onchange="update_search()">
+                                          <option value="run">Run</option>
+                                          <option value="gtid">GTID</option>
+                                          <option value="date">Date</option>
+                                      </select>
+                                  </div>
+                                  <label class="col-sm-1 control-label">
+                                      Start
+                                  </label>
+                                  <div class="col-sm-2">
+                                      <input name="start"
+                                       id="start"
+                                       type="number"
+                                       min=0
+                                       class="form-control">
+                                  </div>
+                                  <label class="col-sm-1 control-label">
+                                      End
+                                  </label>
+                                  <div class="col-sm-2">
```

Comment:
It would be nice if the start and end dates were auto-filled from the previous search. In other words, right now if I pick a start and end date and then search, when the page reloads these are lost and it defaults back to searching by run. This can be annoying if you search by a particular date or run range and then just want to change a single day.

Reply:
done.
(hate javascript)